.jenkins/caffe2/build.sh

```diff
@@ -134,16 +134,59 @@ build_args+=("BUILD_TEST=ON")
 build_args+=("INSTALL_TEST=ON")
 build_args+=("USE_ZSTD=ON")
 
-if [[ $BUILD_ENVIRONMENT == *py2-cuda9.0-cudnn7-ubuntu16.04* ]]; then
+if [[ $BUILD_ENVIRONMENT == *linux-xenial-cuda9-cudnn7-py3* ]]; then
```

Comment:
do want to still test the integration with tensor5?

Reply:
TRT5 integration is still there but it's not (and can't be) covered by extensive test. TRT6 does support all we need, so we are trying to find an environment where it would run.

Reply:
Try 'py3.5-cuda10.1-cudnn7-ubuntu16.04'?

https://github.com/pytorch/ossci-job-dsl/blob/master/src/main/groovy/ossci/caffe2/Images.groovy#L26

We don't enabled caffe2 py3 build/test instance on PR. We should enable one. @yf225, Could you help on this?